### ISSUE TITLE
Emote licensing

### ISSUE THREAD

**Issue by FFY00** [bimbo-garden/greenhouse#16]
How should we handle licensing?

I think we should have a license field on the upload page, and probably select [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) by default.

For emotes with unknown creators, we should have an "Unknown" field.

**Comment by IZeeLikeEyeZee** [comment]
Yeah I think default behaviour should be twitch account selection, with an unknown option, and options to add other links to the artist. Also a field to just type whatever if for example you only know the discord username of the creator or smth like that.

As for licensing I say fuck IP, emotes are like language and art in that they evolve and are all derivative, but if we have to have a license field I think that's a good default option. Ngl tho I don't know all that much about licensing and how it relates to IP and stuff so Devin or Matt might be better people to ask about this, since they have a lot of experience and don't have the same capitalist brainrot that normies might.
